### ISSUE TITLE
Support to other cooperative perception dataset

### ISSUE THREAD

**Issue by xinchundong** [coperception/coperception#8]
Hi Team,

Thanks for your great work! Do you have plans to support other cooperative perception dataset such as OPV2V (ICRA2022) , V2X-ViT's V2XSet (ECCV2022), and CODD(RAL) as well?

**Comment by simbaforrest** [comment]
Hi @xinchundong , yes, we are discussing this with the UCLA team. If you are interested, feel free to join this effort and contribute directly to this repository. Our goal is to build a community and avoid re-inventing the wheels.

@RoboticsYimingLi @dekunma @ShunliRen FYI.

**Comment by xinchundong** [comment]
I really look forward to this! With an unified framework, we do not need to switch to different repos anymore....

**Comment by simbaforrest** [comment]
Exactly, and this repo's aim is to build a community to collaboratively develop and maintain all new methods and datasets in the CoPerception domain. 

So you are more than welcome to build your new methods on this codebase (which could be merged into the master branch when you are ready to).
